package.json

```diff
@@ -19,7 +19,7 @@
     "lint": "eslint --ext .ts src/",
     "lint-fix": "eslint --ext .ts src/ --fix",
     "pretest": "yarn run check-types",
-    "test:unit": "nyc -r lcov -e .ts mocha -r ./.babel-register 'test/unit/**/*.test.ts' && nyc report",
+    "test:unit": "nyc -r lcov -e .ts mocha -r ts-node/register 'test/unit/**/*.test.ts' && nyc report",
```

Comment:
How come this is changed?

Reply:
Oh, I didn't read the description xD

Reply:
I was sure, somebody will ask but yeah it enables us to stub out imported functions:)